app/policies/comment_policy.rb

```diff
@@ -39,6 +39,10 @@ def unhide?
     user_is_commentable_author?
   end
 
+  def admin_delete?
+    user_is_admin?
```

Comment:
This can be refactored to use the inherited method `user_admin?` from `application_policy.rb`.